100-level-AsyncAPI-Intro/step1.md

```diff
@@ -0,0 +1,61 @@
+
+Chan is now curious about how AsyncAPI can help develop event-driven architecture.
+
+## Event-driven architecture with AsyncAPI
+
+"AsyncAPI simplifies the definition of communication and message exchange among different components in an Event-Driven Architecture (EDA). It provides a structured way to describe events, channels, and message formats, enhancing overall understanding.
+
+AsyncAPI simplifies the definition of communication and messages between different components. AsyncAPI offers a structured way to describe the events, channels, and message formats.
+
+AsyncAPI helps you define asynchronous APIs. But what is an asynchronous API?
+
+They are like digital mailboxes where you can send messages without waiting for an instant reply. When the other person reads your letter and has something to say, they will reply. 
+
+Async API helps map out events and their response in an EDA.
+
+Chan spots cake in Eve's kitchen and asks for a slice as she likes cakes. While offering her a slice, Eve gives her an example: imagine a birthday party with events like cake cutting and gift opening. These events trigger responses, such as singing during the cake cutting or clapping while opening the gifts.
+
+AsyncAPI helps organize events, determining when they'll occur and how everyone should react. For example, during the cake cutting, AsyncAPI guides in singing the birthday song. It also enables the creation of a party schedule, like cutting the cake at 6 pm and opening gifts at 7 pm.
+
+Here are reasons why using AsyncAPI enhances EDA design:
+
+* AsyncAPI plans and describes each event and it corresponding reactions across different components.
+* AsyncAPI facilitates communication between different components for seamless information exchange.
+* AsyncAPI allows components to complete tasks independently without waiting.
+
+Eve further assists Chan in understanding how AsyncAPI can enhance communication between microservices."
+
+## Microservices with AsyncAPI
+
+In a microservices architecture, each microservice represents a specific function. Different microservices communicate using events, much like teams collaborating on tasks. For example, one team handles baking the birthday cake, while another focuses on decorating the room. These teams must communicate to share their progress.
+
+AsyncAPI helps different microservices communicate with each other. For microservices to talk to each other, they would have to:
+
+* Define how to send and receive messages
+* Schedule when to send these messages
+* Ensure consistent language in messages
+
+AsyncAPI ensures that each microservice knows what information it will receive, how it will receive it, and what it should do with it. AsyncAPI helps document interactions between microservices for developers developer understanding and integration.
+
+Now, with coffee in hand, Eve and Chan are ready to step into Chan's spaceship to enhance its communication system.
```

Comment:
```suggestion
In a microservices architecture, each microservice represents a specific function. Different microservices communicate using events, similar to how teams collaborate on tasks. For example, one team handles baking the birthday cake, while another focuses on decorating the room. These teams must communicate to share their progress.

For AsyncAPI to help different microservices communicate with each other, they must:

* Define how to send and receive messages
* Schedule when to send these messages
* Ensure consistent language in messages

AsyncAPI ensures that each microservice knows what information it will receive, how it will receive, and what it should do with it. 

Now, with coffee in hand, Eve and Chan are ready to step into Chan's spaceship to enhance its communication system.
```